backend/wordfactory-preprocessed.json

```diff
@@ -162527,7 +162527,7 @@
       "lessonIndex": 1,
       "lessonPrefix": "Les 1",
       "lessonInstruction": "Morfemen zijn de kleinste stukjes taal die een betekenis hebben. Bijvoorbeeld sterkste bestaat uit de stam sterk en –ste, wat aangeeft dat iets of iemand niet gewoon sterk is, maar het meest sterk.\n\nJe kunt morfemen herkennen door te analyseren welke stukjes bijdragen aan de betekenis van een woord. Iemand kan sterk zijn zonder de sterkste te zijn. Sterkste betekent dus iets anders dan sterk, wat betekent dat –ste een morfeem is.",
-      "lessonTitle": "Instructie",
+      "lessonTitle": "Les2: Beginnersniveau I",
```

Comment:
Dit was een typfout van Josien, maar krijg je nu niet dubbel 'les' te zien in de front-end? Denk dat je hier en zowel alleen `Beginnersniveau` neer moet zetten en `II` ipv `I`

Reply:
^